frappe/commands/site.py

```diff
@@ -1274,9 +1296,30 @@ def handle_data(data: dict, format="json"):
 		data = [["DocType", "Fields"]] + [[table, ", ".join(columns)] for table, columns in data.items()]
 		render_table(data)
 
+def add_new_user(
+    email, first_name=None, last_name=None, send_welcome_email=False, password=None,role=None):
+    # add user
+    user = frappe.new_doc("User")
+    user.update(
```

Comment:
Your function name is add_new_user
So when I read ".update," I think you need to update user data, but you need to add a new user

So I prefer to use the dictionary in these cases 
you can use this block of code 

user = frappe.get_doc({
			"doctype": "User",
            "email": email,
            "enabled": 1,
            "first_name": first_name or email,
            "last_name": last_name,
            "user_type": "System User",
            "send_welcome_email": 1 if send_welcome_email else 0,
})
user.insert()

The two-way is working well but I think the second is because I see that more readable
I'm waiting for Your opinion
Thank you for your time and code 😊